docs/automated_testing.md

```diff
@@ -470,7 +470,25 @@ The `continue` Step will tap the button to continue to the next page. The text o
 
 ---
 
-Use the `set value` Step to set the values of fields.
+You can also tap or click on specific elements, like buttons on a page.
+You can use any valid [CSS Selector](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors)
+to get an element on the page, and can add any additional wait time after tapping the element.
+
+```
+    When I tap the "#element-id" element
+    And I tap the "#other-element" element and wait for 5 seconds
+```
+
+If you are using the [ALToolbox tabs](framework/altoolbox.md#display-a-series-of-tabs), you can say that you're tapping the tab,
+and Kiln will tap and wait until the tab is fully visible.
```

Comment:
I've been trying to be more consistent with the name. I'm not sure that it's necessary or even desirable. I'm sometimes concerned about the confusion of tying it too closely to AssemblyLine. I'd be fine if you left it as it is, but offering this so it can be a deliberate choice.

```suggestion
and ALKiln will tap and wait until the tab is fully visible.
```